### ISSUE TITLE
Loss Function

### ISSUE THREAD

**Issue by tennpalight** [ifzhang/FairMOT#338]
Thank you for your  work and code! 
I have a question about the loss function.
The loss function is defined in the paper and code as follows
`loss = torch.exp(-self.s_det) * det_loss + torch.exp(-self.s_id) * id_loss + (self.s_det + self.s_id) `
`loss *= 1/2`
I know that id_loss is cross entropy loss and det_loss is the weighted sum of the the loss functions of the detectors.
However, according to "Multi-task learning using uncertainty to weigh losses for scene geometry and semantics" ,
the uncertainty loss in case cross entropy loss is exp(-s)*(cross entropy loss)
Therefore, shouldn't the Loss function be defined as 
`loss = (1/2)*torch.exp(-self.s_det) * det_loss + torch.exp(-self.s_id) * id_loss + (1/2)*(self.s_det + self.s_id)`
or
`loss = torch.exp(-self.s_det) * det_loss + torch.exp(-self.s_id) * id_loss + (1/2)*(self.s_det + self.s_id)`
I would appreciate your feedback.

**Comment by HanGuangXin** [comment]
Actually, I don't think it matters. Because the weight parameter will update during training procedure.

**Comment by tennpalight** [comment]
Thanks for the answer!
I have done some experiments based on your program. As a result, the initial value  and the  magnitude of the coefficients has little effect.
I think you're right in practical terms.
However, I am  concerned because there was no mention of it in your paper or in your program
